src/frontend/components/cartProductCard/CartProductCard.js

```diff
@@ -0,0 +1,112 @@
+import { useProduct } from "../../context/index";
+import { useState, useEffect } from "react";
+import { updateBillDetails, updateQuantity } from "../../utils/cartBill/index";
```

Comment:
Try to use Absolute Imports to avoid `../../` else it will be prone to mistakes. You can read about them here: https://create-react-app.dev/docs/importing-a-component/#absolute-imports